### ISSUE TITLE
Create home page for user

### ISSUE THREAD

**Issue by avenmia** [kmal808/go-or-no#2]
Present some basic data to the user:

* Grant permissions to location
* Show a Go or No Go
  * Determined by: 
    * Swell direction (not periods for novices)
    *  Wave height
    *  Wind
* Give an ELI5 surf report to the user

Smaller Tasks:
*  Get a successful api call for swell direction ,wave height, and wind
*  Define novice, intermediate, and experience "metrics" (For example, novice could be wave heights <= 3 feet)
*  Display a "Go" or "No Go" based on the retrieved data

Future Considerations:
* Customized settings for the user may want to see? 
  * For example, experienced surfers may want swell periods. This could be a toggle in the user settings

**Comment by kmal808** [comment]
Adding on to the future considerations: 
- Displaying an embedded map
- Map data could include:
- - Users current location
- - Pin or icon on all breaks within the current map window
- - An onClick on the pin/icon to show a "quick glance" card containing relevant info

**Comment by avenmia** [comment]
API: https://magicseaweed.com/docs/developers/59/querying-the-api/9909/
